src/components/catalog/list.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```

Comment:
Styled component shouldn't be in a separate file with all styled components ?

Reply:
No.. I don't see any profit in it.